README.md

```diff
@@ -61,3 +61,74 @@ docker build -t astral:latest .
 ```
 docker run -p 3000:3000 astral:latest
 ```
+
+# OIDC configuration
+The OIDC modules allow the assignment of a policy to an OIDC user, by
+mapping that user's email address to a policy we create.  They work as
+follows:
+
+OidcProvider::configure_as_oidc_provider() creates an OIDC provider
```

Comment:
I think this is `OidcProvider.new.configure` now

Reply:
doh!  thanks!